### DISCUSSION TITLE
VS Code useful extensions VS_Code_Extensions_ReadMe.md

### DISCUSSION THREAD

**Discussion by Tezei** [Cyfrin/foundry-full-course-cu#2855] [Ideas]
Hiya guys,
I am suggesting a small useful idea to make a **VS_Code_Extensions_ReadMe.md** file with a useful extensions with some general shortcuts to be used.

I am suggesting [**Colonize**](https://marketplace.visualstudio.com/items?itemName=vmsynkov.colonize) extension or other to use for a semicolon at the end of line.

10x for the awesome work!

Best Regards!

**Comment by EngrPips** [comment]
Thanks for the suggestion, Friend.

**Reply by Tezei** [reply]
No problem, glad to help!. :)
